test/onnx/test_onnx_common.py

```diff
@@ -38,6 +39,11 @@ def _run_model_test(test_suite: _TestONNXRuntime, *args, **kwargs):
     return verification.verify(*args, **kwargs)
 
 
+def parameterize_class_name(cls: Type, idx: int, input_dicts: Mapping[Any, Any]):
```

Comment:
I would add a docstring to this function to make it clear how anyone can use it. especially since it is shared